### PR TITLE
Update requests-click session for B191

### PR DESCRIPTION
**Pull request by MarekSuchanek**

### CODE REVIEW

File: lessons/intro/click/index.md [hunk 1]
```diff
@@ -188,8 +196,42 @@ def cat(files):
...
+můžete použít vyjímku [`click.UsageError`](https://click.palletsprojects.com/en/7.x/api/#click.UsageError)
+nebo [`click.BadParameter`](https://click.palletsprojects.com/en/7.x/api/#click.BadParameter)
+(vztahuje-li se přímo ke konkrétnímu parametru). Click se pak 
+sám postará o případné ukončení programu s odpovídající chybovou
+hláškou a kódem.
+
+```python
+def validate_username(ctx, param, value):
+    try:
+        assert len(value) == 8
```
Comment (hroncok) [inline lessons/intro/click/index.md]: [Don’t Use Asserts for Data Validation](https://dbader.org/blog/python-assert-tutorial)

Comment (hroncok) [inline lessons/intro/click/index.md]: Not important, but some usernames can be shorter.

```suggestion
        assert 2 <= len(value) <= 8
```

**Review by hroncok — CHANGES_REQUESTED** [review]
Looks good apart form the validation asserts. Thanks.

File: lessons/intro/click/index.md (hunk 1 above)
Comment (MarekSuchanek) [reply lessons/intro/click/index.md]: Fixed in https://github.com/cvut/naucse.python.cz/pull/12/commits/5f3035c992fc3bf58ed47b3489f4ebef0e45505f

Comment (MarekSuchanek) [reply lessons/intro/click/index.md]: Fixed in https://github.com/cvut/naucse.python.cz/pull/12/commits/5f3035c992fc3bf58ed47b3489f4ebef0e45505f

**Review by hroncok — APPROVED** [review]

**Pull request merged**